src/main/resources/application.yaml

```diff
@@ -45,6 +48,8 @@ spring:
           auth: true
           starttls:
             enable: true
+  main:
+    allow-bean-definition-overriding: true
```

Comment:
what's left that requires this?

Reply:
for application itself just auth config. integration tests need further investigation. i'll move that to integration config and apply fix to auth config ;)